src/sidebar/test/bootstrap.js

```diff
@@ -1,6 +1,22 @@
 // Expose the sinon assertions.
 sinon.assert.expose(assert, { prefix: null });
 
+assert.rejects = async (asyncFn, error) => {
```

Comment:
I was slightly confused initially as here `asyncFn` is actually a _promise_ rather than a function that returns a Promise, which is what the name `asyncFn` implies to me.